Extension/lib/content-script/preload.js

```diff
@@ -57,7 +57,8 @@
         // We use shadow DOM when it's available to minimize our impact on web page DOM tree.
         // According to ABP issue #452, creating a shadow root breaks running CSS transitions.
         // Because of this, we create shadow root right after content script is initialized.
-        var shadowRoot = document.documentElement.shadowRoot;
+        // First check if it's available already

```

Comment:
You should alert reader, that browser will print warning if you try to create additional root.